packages/space-opera/src/components/camera_settings/reducer.ts

```diff
@@ -98,12 +96,6 @@ export function dispatchCameraTarget(target?: Vector3D) {
   return {type: SET_CAMERA_TARGET, payload: target};
 }
 
-/** Dispatch initial orbit in camera state */
-const SET_CAMERA_STATE_INITIAL_ORBIT = 'SET_CAMERA_STATE_INITIAL_ORBIT';
-export function dispatchInitialOrbit(orbit: SphericalPositionDeg|undefined) {
```

Comment:
the dispatchInitialCameraOrbit was a duplicate that was performing the same state change as dispatchSaveCameraOrbit, so it was repealed and replaced